SLHCUpgradeSimulations/Configuration/python/HCalCustoms.py

```diff
@@ -93,7 +93,13 @@ def customise_Hcal2017(process):
         process=customise_mixing(process)
     if hasattr(process,'simHcalTriggerPrimitiveDigis'):
         process.simHcalTriggerPrimitiveDigis.upgradeHF = cms.bool(True)
-    
+    if hasattr(process,'dqmoffline_step'):
```

Comment:
Yes, HCalCustoms is just being used for development right now. Hopefully everything will be moved to Eras by pre12.